projectTemplate/Microsoft.AzureFunctions.ProjectTemplates/content/AzureFunctionsProjectTemplate/HttpTriggerCSharp.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Linq;
```

Comment:
The project template is supposed to be empty, Remove the HttpTriggerCSharp.cs file